examples/deployments/do-terraform/variables.tf

```diff
@@ -0,0 +1,139 @@
+variable "instance_image" {
+    description = "The image to use for the instance"
+    type        = string
+    default     = "ubuntu-22-04-x64"
+}
+variable "chroma_release" {
+  description = "The chroma release to deploy"
+  type        = string
+  default     = "0.4.12"
+}
+
+#TODO this should be updated to point to https://raw.githubusercontent.com/chroma-core/chroma/main/examples/deployments/common/startup.sh in the repo
```

Comment:
We can update it in this PR right? Since they'll be updated simultaneously

Reply:
At the time of dev/test this was not present, hence the TODO.

Reply:
Makes sense -- thank you for updating!